collect_app/build.gradle

```diff
@@ -160,6 +160,12 @@ dependencies {
     androidTestCompile group: 'com.android.support', name: 'support-annotations', version: '25.1.0'
     androidTestCompile group: 'com.android.support.test', name: 'runner', version: '0.5'
     androidTestCompile group: 'com.android.support.test', name: 'rules', version: '0.5'
+
+    // firebase core library
+    compile 'com.google.firebase:firebase-core:9.0.0'
```

Comment:
Can you please match the surrounding style and use the standard long-form for the dependency rather than the shorthand? https://docs.gradle.org/current/userguide/artifact_dependencies_tutorial.html#sec:external_dependencies_tutorial Thanks!